docs/General/Manuals/client-manual.rst

```diff
@@ -1362,10 +1362,43 @@ view will tell you what was sold where.
 Tax Rates View
 ^^^^^^^^^^^^^^
 
+The :guilabel:`Tax Rates View` is simply an icon on the `Top Function Bar`_ that shows you what percentage
+Gold, Science and Luxury Goods your nation is set at. Figure 38 is a sample of the icon for
+:guilabel:`Tax Rates View`.
+
+.. figure:: ../../_static/images/gui-elements/top-bar-tax-rates.png
+  :align: center
+  :alt: Freeciv21 Tax Rates View
+  :figclass: align-center
+
+  Figure 38: Tax Rates View Icon
+
+
+Clicking on the :guilabel:`Tax Rates View` icon will bring up the :guilabel:`Tax Rates Dialog` box as shown in
+Figure 17 in the `Civilization Menu`_ section.
 
 National Status View
 ^^^^^^^^^^^^^^^^^^^^
 
+The :guilabel:`National Status View` is simply an icon on the `Top Function Bar`_ with 4 smaller icons inside of
```

Comment:
Consider rephrasing to "The National Status View on the Top Function Bar shows various information about your nation and the world via icons."